spec/backgrounder/support/backends_spec.rb

```diff
@@ -34,9 +34,92 @@ module CarrierWave::Backgrounder
         let(:args) { ['FakeClass', 1, :image] }
 
         it 'invokes perform_later with string arguments' do
-          expect(MockWorker).to receive(:perform_later).with('FakeClass', '1', 'image')
+          expect(MockActiveJob).to receive(:perform_later).with('FakeClass', '1', 'image')
           mock_module.backend :active_job
-          mock_module.enqueue_for_backend(MockWorker, *args)
+          mock_module.enqueue_for_backend(MockActiveJob, *args)
+        end
+
+        describe 'queue name configuration' do
```

Comment:
In general, I am leaning into integration specs vs mocking/stubbing. I find it harder to maintain, and it relies on mocks being set up correctly as well as library apis never changing... which is never the case. 

I am happy to merge this PR if you can find a way to use the integration specs.